files/hermit

```diff
@@ -4,6 +4,10 @@
 
 set -eo pipefail
 
+# Set Github Token only in CI (Buildkite)
```

Comment:
I don't think this makes sense to be in the open source repo... it's a) Buildkite specific and b) squareup specific.